cli/src/pcluster/constants.py

```diff
@@ -249,6 +250,7 @@ class Feature(Enum):
     Feature.SLURM_DATABASE: [],
     Feature.CLUSTER_HEALTH_METRICS: ["us-iso"],
     Feature.CAPACITY_BLOCK: ["us-iso"],
+    Feature.NLB_SECURITY_GRROUP: ["us-iso"],
```

Comment:
We should cover this in the unit test `test_Feature_validators.py::test_feature_region_validator` and `test_utils.py::test_is_feature_supported`

Reply:
Done